tests/functional/u/unspecified_encoding_py38.txt

```diff
@@ -27,3 +27,5 @@ unspecified-encoding:149:0:149:23::Using open without explicitly specifying an e
 unspecified-encoding:152:0:152:28::Using open without explicitly specifying an encoding:UNDEFINED
 unspecified-encoding:155:0:155:26::Using open without explicitly specifying an encoding:UNDEFINED
 unspecified-encoding:158:0:158:35::Using open without explicitly specifying an encoding:UNDEFINED
+bad-open-mode:161:0:161:25::"""%s"" is not a valid mode for open.":UNDEFINED
```

Comment:
Ouch, that's not a good message! 🤕 

```
a.py:1:0: W1501: "%s" is not a valid mode for open. (bad-open-mode)
```

Track separately?

Reply:
Oh wow, I missed that. Hmm, yeah. Should probably be trivial to fix, but should indeed go in a different issue/PR.